gems/pending/fs/modules/LinuxMountProbe.rb

```diff
@@ -18,7 +18,7 @@ def self.probe(fs)
 
     LINUX_ROOT_FILES.each do |f|
       unless fs.fileExists?(f)
-        $log.debug "LinuxMountProbe << FALSE because file #{f} does not exist" if $log
+        $log.error "LinuxMountProbe << FALSE because file #{f} does not exist" if $log
```

Comment:
This shouldn't be an error. This would log an error for every non-root volume we encounter - info would be a better choice.